### PR TITLE
Release 4.0.0-beta.0

### PR DESCRIPTION
**Pull request by johnhwhite**
- Created Action Hub template component. [#1](https://github.com/blackbaud/skyux-pages/pull/1)

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-pages?_sp.spa=pr-release-4.0.0b0](https://developer.blackbaud.com/skyux-pages?_sp.spa=pr-release-4.0.0b0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
-# x.x.x (yyyy-mm-dd)
+# 4.0.0b0 (2021-04-30)
+
+- Created Action Hub template component. [#1](https://github.com/blackbaud/skyux-pages/pull/1)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]:
```suggestion
- Created the action hub component. [#1](https://github.com/blackbaud/skyux-pages/pull/1)
```
Based on the docs at https://github.com/blackbaud/skyux-pages/pull/4, I assume we don't need `template` as part of the component name. Am I interpreting that correctly?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-pages?_sp.spa=pr-release-4.0.0b0](https://developer.blackbaud.com/skyux-pages?_sp.spa=pr-release-4.0.0b0)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/pages",
-  "version": "0.0.1",
+  "version": "4.0.0b0",
```
Comment (Blackbaud-TrevorBurch) [inline package.json]: We'll want this to be `4.0.0-beta.0`.

We have some pipeline things that look for that specific beta annotation.

**Review by Blackbaud-TrevorBurch — CHANGES_REQUESTED** [review]

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-pages?_sp.spa=pr-release-4.0.0b0](https://developer.blackbaud.com/skyux-pages?_sp.spa=pr-release-4.0.0b0)

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**